<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitManagerClass</class>
 <widget class="QMainWindow" name="GitManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>推送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="commitText">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;仓库数：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>远程仓库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pullButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>拉取</string>
    </property>
   </widget>
   <widget class="QLabel" name="followerNumLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="localReposList">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>141</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="remoteReposList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="lookOverButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addLocalRepoButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本地仓库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginNameLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="repoNumLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshRemoteButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;粉丝数：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLabel" name="headImageLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="promptLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>信息不能为空</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>仓库信息</string>
    </property>
    <addaction name="localReposAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>切换</string>
    </property>
    <addaction name="githubAction"/>
    <addaction name="giteeAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="localReposAction">
   <property name="text">
    <string>本地仓库</string>
   </property>
  </action>
  <action name="githubAction">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="giteeAction">
   <property name="text">
    <string>Gitee</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
